doc/interface/binding/multi_component_colloidal.rst

```diff
@@ -0,0 +1,163 @@
+===================  =========================  =========================================
+
+``COL_LOGKEQ_SALT_EXPFAC``
+   Protein-resin interaction :math:`K_{e,i}` equilibrium: Constant pre-factor :math:`k_{c,i}` for e-function with salt power
+
+===================  =========================  =========================================
+**Type:** double     **Range:** :math:`\ge 0`   **Length:** NTOTALBND
+===================  =========================  =========================================
+
+``COL_LOGKEQ_SALT_EXPARGMULT``
+   Protein-resin interaction :math:`K_{e,i}` equilibrium: Constant power factor :math:`k_{d,i}` for salt in e-function
+
+===================  =========================  =========================================
+**Type:** double     **Range:** :math:`\ge 0`   **Length:** NTOTALBND
+===================  =========================  =========================================
+
+``COL_BPP_PH_EXP``
+   Protein-protein interaction :math:`b_{pp,i}`: Constant power term :math:`b_{e,i}` for pH.
+   If pH is not considered, this value will be not be used but must still be specified, i.e. can be any number.
+
+===================  =========================  =========================================
+**Type:** double     **Range:** :math:`\ge 0`   **Length:** NTOTALBND
+===================  =========================  =========================================
+
+``COL_BPP_SALT_POWFACT``
+   Protein-protein interaction :math:`b_{pp,i}`: Constant power pre-factor :math:`b_{a,i}` for salt
+
+===================  =========================  =========================================
+**Type:** double     **Range:** :math:`\ge 0`   **Length:** NTOTALBND
+===================  =========================  =========================================
+
+``COL_BPP_SALT_POWEX``
+   Protein-protein interaction :math:`b_{pp,i}`: Constant power :math:`b_{b,i}` for salt
+
+===================  =========================  =========================================
+**Type:** double     **Range:** :math:`\ge 0`   **Length:** NTOTALBND
+===================  =========================  =========================================
+
+``COL_BPP_SALT_EXPFACT``
+   Protein-protein interaction :math:`b_{pp,i}`: Constant pre-factor :math:`b_{c,i}`  e-function with salt power
+
+===================  =========================  =========================================
+**Type:** double     **Range:** :math:`\ge 0`   **Length:** NTOTALBND
+===================  =========================  =========================================
+
+``COL_BPP_SALT_EXPARGMULT``
+   Protein-protein interaction :math:`b_{pp,i}`: Constant power factor :math:`b_{d,i}` for salt in e-function 
+
+===================  =========================  =========================================
+**Type:** double     **Range:** :math:`\ge 0`   **Length:** NTOTALBND
+===================  =========================  =========================================
+
+``COL_RADIUS``
```

Comment:
Suggestion: Rename to `COL_PROTEIN_RADIUS` to avoid confusion with column inner radius.

Reply:
agreed and done